lib/internal/blob.js

```diff
@@ -312,6 +312,9 @@ class Blob {
     return dec.decode(await this.arrayBuffer());
   }
 
+  /**
```

Comment:
would you mind reverting the changes that is not related to this PRs and only leave the JSDoc part in?

Reply:
sure, I'll fix it asap :)